ukvm/ukvm.h

```diff
@@ -142,4 +142,55 @@ extern struct ukvm_module ukvm_module_blk;
 extern struct ukvm_module ukvm_module_net;
 extern struct ukvm_module ukvm_module_gdb;
 
+/*
+ * GDB specific functions to be implemented on all backends for all
+ * architectures.
+ */
+
+/*
+ * Fills *registers with a stream of hexadecimal digits for each register in
```

Comment:
`*registers` -> `*reg`. Also, this should say "for each guest register" or something like that.